apps/src/netsim/NetSimNode.js

```diff
@@ -103,16 +89,7 @@ NetSimNode.prototype.getNodeType = function () {
  * @returns {string}
  */
 NetSimNode.prototype.getStatus = function () {
```

Comment:
Good catch.  I think that comment was outdated even before this PR; I'm no longer using a status enum at all.  I've updated the comment and made the base implementation of this method throw an error (as `getNodeType()` does).